### PR TITLE
Change endpoints to use CursorPagination

### PR DESCRIPTION
**Pull request by erikh360**

### CODE REVIEW

**Comment by erikh360** [comment]
Hold on, let me do the dashboards part first.

File: dashboards/tests.py [hunk 1]
```diff
@@ -1 +1,52 @@
...
+
+
+class TestServicesApp(AuthenticatedAPITestCase):
+
+    def test_list_dashboard(self):
+
+        response = self.adminclient.get('/api/v1/dashboard/')
+
+        body = response.json()
+        self.assertEqual(len(body['results']), 0)
```
Comment (KaitCrawford) [inline dashboards/tests.py]: why not add an object or two to make these tests more useful?

File: dashboards/tests.py [hunk 2]
```diff
@@ -1 +1,52 @@
...
+
+    def test_list_userdashboard(self):
+
+        response = self.adminclient.get('/api/v1/userdashboard/')
+
+        body = response.json()
+        self.assertEqual(len(body['results']), 0)
```
Comment (KaitCrawford) [inline dashboards/tests.py]: as above, maybe add an object or two here?

File: dashboards/tests.py [hunk 3]
```diff
@@ -1 +1,52 @@
...
+
+    def test_list_definition(self):
+
+        response = self.adminclient.get('/api/v1/definition/')
+
+        body = response.json()
+        self.assertEqual(len(body['results']), 0)
```
Comment (KaitCrawford) [inline dashboards/tests.py]: and here?

**Review by KaitCrawford — COMMENTED** [review]
I think you could make a few of the tests a bit more useful by adding some objects to list

**Comment by erikh360** [comment]
This is ready for another look @KaitCrawford

**Review by KaitCrawford — APPROVED** [review]
Thanks @erikh360 :+1:

**Pull request merged**